soroban-sdk/src/testutils.rs

```diff
@@ -169,6 +169,9 @@ pub trait Accounts {
 
     /// Remove an account.
     fn remove(&self, id: &crate::AccountId);
+
+    /// Updates balance
```

Comment:
This comment is pretty useless, could you please extend it? I think we should mention that this is native token balance of the classic account.

Reply:
Done